core/src/lib.rs

```diff
@@ -291,12 +339,29 @@ impl Schema {
     pub fn identifies_attribute(&self, x: &NamespacedKeyword) -> bool {
         self.get_entid(x).map(|e| self.is_attribute(e)).unwrap_or(false)
     }
+
+    /// Returns an symbolic representation of the schema suitable for applying across Mentat stores.
+    pub fn to_edn_value(&self) -> edn::Value {
+        edn::Value::Vector((&self.schema_map).into_iter()
```

Comment:
There are two different ways to get an iterator: `into_iter` consumes the container, giving you an iterator over `T`, and `iter`, which _doesn't_ consume the container, but gives you an iterator over `&T`.

In general, use `into_iter` to consume temporary values or transform sequences. Use `iter` when you just want to peek at the contents.